src/selfplay/game.cc

```diff
@@ -91,19 +97,39 @@ SelfPlayGame::SelfPlayGame(PlayerOptions white, PlayerOptions black,
     tree_[1] = std::make_shared<NodeTree>();
     tree_[1]->ResetToPosition(orig_fen_, {});
   }
+  int ply = 0;
+  auto white_prob = white.uci_options->Get<float>(kOpeningStopProbId);
+  auto black_prob = black.uci_options->Get<float>(kOpeningStopProbId);
+  if (white_prob != black_prob && white_prob * black_prob > 0.0f) {
+    throw Exception("Stop probabilities must be both equal or zero!");
+  }
+
   for (Move m : opening.moves) {
+    auto exit_prob_now = tree_[0]->IsBlackToMove() ? black_prob : white_prob;
+    auto exit_prob_next = tree_[0]->IsBlackToMove() ? white_prob : black_prob;
+    int positions = opening.moves.size() - ply + 1;
```

Comment:
I believe it was originally named something like that, however we don't count the number of plies (i.e. half-moves), but we count the number of exit positions. If the variable was named anything `ply`, this might lead to confusion over the +1. However, obviously this variable name isn't doing a better job at conveying the meaning, so suggestions are welcome.